### DISCUSSION TITLE
How to retrieve the current version?

### DISCUSSION THREAD

**Discussion by JohnVerheij** [danielpalme/ReportGenerator#713] [Q&A] [answered]
Using `reportgenerator -v` does not work. It gives the following output:

```
2024-12-05T09:49:56: Arguments
2024-12-05T09:49:56:  -v
2024-12-05T09:49:56: No report files specified.
2024-12-05T09:49:56: No target directory specified.
```

I just want to check if it is installed and to print the installed version. How can I achieve this?

**Comment by danielpalme** [comment, marked as answer]
You can use `reportgenerator -h`.

**Reply by JohnVerheij** [reply]
Thanks, this works for me :)
